src/ext/legit/peek-legit.lisp

```diff
@@ -320,7 +320,13 @@ Notes:
 (define-command peek-legit-select () ()
   (alexandria:when-let ((file (get-matched-file)))
     (quit)
-    (alexandria:when-let ((buffer (find-file-buffer file)))
+    (alexandria:when-let
+        ((buffer (or (and (uiop:file-exists-p file)
```

Comment:
(I don't really know the context for this fix)

In legit, pressing Enter in the legit window calls this function to visit the file at point. There was a file listing, the file should exist. So if the file doesn't exist anymore, we maybe should display a message to the user (instead of doing nothing)? (and refresh the legit window?)

Also I thought the `%filename` slot of a buffer had the full path, not requiring to get the project root?

exple:

    [ ]  %FILENAME       = "/home/vince/lisp-projects/lem/legit.lisp"

Reply:
I'm not sure, but seems like the path is indeed not correct:

https://github.com/lem-project/lem/assets/26670956/837283d4-a1a2-4388-aebc-03184462c592

Maybe the buffer doesn't have the information about the absolute path? :thinking:

Reply:
I can't reproduce. Do you try to visit an unstaged file?

- start from directory mode
- move cursor on an "unstaged changes" line
- press Enter

? this works for me, I visit the file at point. I can stage it with "s" too.

Reply:
Yes I did just that :thinking: